<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWidget</class>
 <widget class="QWidget" name="ImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scroll_area">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>670</width>
          <height>163</height>
         </rect>
        </property>
        <widget class="QLabel" name="image_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>671</width>
           <height>171</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="zoom_in_button">
         <property name="text">
          <string>Przybliż</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zoom_out_button">
         <property name="text">
          <string>Oddal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="open_button">
         <property name="text">
          <string>Otwórz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
